.gitlab-ci.yml

```diff
@@ -320,18 +346,18 @@ spack-scr-lassen:
     - echo "CI_PIPELINE_IID is $CI_PIPELINE_IID"
     - echo "BUILD_DIR_BASE is $BUILD_DIR_BASE"
     - echo "Working dir " `pwd`
-#    - git clone -b robertkb-fix-scr-dependencies https://github.com/robertkb/spack.git
     - rm -rf spack
     - git clone https://github.com/spack/spack.git
```

Comment:
Another spot, let's see if ``--depth 1`` works, and anywhere else we do a ``git clone``